v2/apiserver/schemas/project-role-assignment.json

```diff
@@ -0,0 +1,83 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
+	"$id": "github.com/brigadecore/brigade/v2/project-role-assignment.schema.json",
+
+	"definitions": {
+
+    "apiVersion": {
+			"type": "string",
+			"description": "The major version of the Brigade API with which this object conforms",
+			"enum": ["brigade.sh/v2"]
+		},
+
+		"kind": {
+			"type": "string",
+			"description": "The type of object represented by the document",
+			"enum": ["RoleAssignment"]
+		}
+
+  },
+
+  "title": "ProjectRoleAssignment",
+	"type": "object",
+	"required": ["apiVersion", "kind", "principal", "role"],
+	"additionalProperties": false,
+	"properties": {
+    "apiVersion": {
+      "$ref": "#/definitions/apiVersion"
+    },
+    "kind": {
+      "$ref": "#/definitions/kind"
+    },
+    "principal": {
```

Comment:
Q: is this `principal` property the same between this and the RoleAssignment schema?  If so, we could potentially place in common.json

Reply:
Yes. Will extract. This another case of the common defs not having been established before I wrote this.